presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupsModule.java

```diff
@@ -27,10 +29,11 @@
     @Override
     public void configure(Binder binder)
     {
-        configBinder(binder).bindConfig(DbResourceGroupConfig.class);
+        configBinder(binder).bindConfig(DbReloadingResourceGroupConfig.class);
+        binder.bind(ReloadingResourceGroupConfig.class).to(DbReloadingResourceGroupConfig.class).in(Scopes.SINGLETON);
```

Comment:
this line doesn't make any sense.

Please read about the general guice - `binder.bind(<interface>).to(<implementation>)` call.